test/continuous.jl

```diff
@@ -98,3 +98,7 @@ end
     @test var(d) ≈ δ * α^2 / g^3
     @test skewness(d) ≈ 3β/(α*sqrt(δ*g))
 end
+
+@testset "edge cases" begin
+    @test cdf(Rayleigh(1), 0) === 0.0  # not -0
```

Comment:
Maybe it's helpful to mention the issue here?
```suggestion
    # issue #1371: cdf should not return -0.0
    @test cdf(Rayleigh(1), 0) === 0.0
```